invenio_rdm_records/services/communities/service.py

```diff
@@ -24,6 +25,38 @@ def create(self, identity, data, record, uow=None):
         pass
 
     @unit_of_work()
-    def delete(self, identity, id_, revision_id=None, uow=None):
+    def delete(self, identity, record_id, data, revision_id=None, uow=None):
         """Remove communities from the record."""
-        pass
+        record = self.record_cls.pid.resolve(record_id)
```

Comment:
if my new service I have also added:
```
communities = data.get("communities", [])
if not communities:
    raise ValidationError(_("No communities provided."), field_name="communities")
```

Reply:
In the marshamallow schema I added the validation of the length which will take care of this, thanks for pointing this out!